demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -161,9 +162,9 @@ const VOICE_FOCUS_SPEC = {
 
 const MAX_VOICE_FOCUS_COMPLEXITY: VoiceFocusModelComplexity | undefined = undefined;
 
-type VideoFilterName = 'Emojify' | 'CircularCut' | 'NoOp' | 'Segmentation' | 'None';
+type VideoFilterName = 'Emojify' | 'CircularCut' | 'NoOp' | 'Segmentation' | 'Resize (9/16)' | 'None';
```

Comment:
I think people often write aspect ratio as 16:9 right? Also can we use 1:1 or something that does not match with any screensize?

Reply:
Yes 16/9 is what is called landscape mode and it is the most common aspect ratio. It is also the default aspect ratio in our meeting demo (almost exact). 

9/16 is a different aspect ratio (most commonly used for mobile phones). 9/16 is better than 1/1 as the change in video stream is significant and it helps showcase the functionality better.

Reply:
yeah but that means it kinda weird if I test it in a mobile phone right? I just think we should choose something that is not popular so no matter what device we test it will change.

Reply:
In the mobile phone, the default aspect ratio is 16/9 currently so adding 9/16 filter is a noticeable change.